### PR TITLE
Personal site

### PR DESCRIPTION
**Pull request by abaibik**

### CODE REVIEW

File: personalSite.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      width: 700px;
+    "
+  >
+    <main style="padding: 2rem">
+      <nav>
+        <ul style="display: flex; gap: 5rem; padding-left: 17rem">
+          <li>Contact</li>
+          <li><a href="#skills">Skills</a></li>
+        </ul>
+      </nav>
```
Comment (alhen002) [inline personalSite.html:29]: The navigation could be  in a <head> element instead

**Review by alhen002 — APPROVED** [review]
Good solution 👍

File: personalSite.html (hunk 1 above)
Comment (abaibik) [reply personalSite.html:29]: What du you mean? :slightly_smiling_face: What kind of element?

Comment (alhen002) [reply personalSite.html:29]: The  nav element could be in a head element above the main.

Comment (abaibik) [reply personalSite.html:29]: Ah so! Thank you 🙂

**Pull request merged**